### DISCUSSION TITLE
RDP to Ubuntu 20.04 via Remmina: "Cannot connect to the "IP" server"

### DISCUSSION THREAD

**Discussion by GeorgiosDolias** [neutrinolabs/xrdp#2735] [Q&A]
Hi everyone,

After a few successful connections up to the login window, I'm trying to connect to a PC with Ubuntu 20.04 via:

- RDP with Remote Desktop (in Windows 10) and 
- Remmina 1.4.29 (Linux Mint 20.1) 

but I haven't managed  to do so since then, I receive the error message: Cannot connect to the "IP" server".
The firewall is inactive.
I have tried all security protocol negotiations (NLA, TLS, RDP) along with the default automatic negotiation. 
I can successfully connect with SSH to the PC via Remmina. 

- The files /var/log/xrdp.log and /var/log/xrdp-sesman.log are not updated whenever I try to connect, but only when the server is restarted.

**Comment by matt335672** [comment]
If the log's not being updated when you try to connect, there's some sort of network issue. If this is a droplet, see #2645 and #2495.

When it fails, try these commands from the Mint machine and see what you get. Make the relevant substitution for `$HOSTNAME`:-
```
nmap -p 3389 $HOSTNAME
nmap -4 -p 3389 $HOSTNAME
nmap -6 -p 3389 $HOSTNAME

```

**Comment by matt335672** [comment]
As you correctly deduce, there is something between the Ubuntu server and the Mint client which is blocking port 3389.

Your options would seem to be:-
1. Find out what is  blocking the port. You might be able to do this with a TCP traceroute using port 3389.
2. Use a different port at both ends
3. Set up a point-to-point VPN between the client and the server.

You seem to be using a public IP for your server, so I'd really recommend this option as being the most secure.

Also, I've edited your post above to remove sensitive information. If you're able, I suggest you delete the original version of your post. I don't see able to do it from here.
